### DISCUSSION TITLE
Server not scanning VST plugins

### DISCUSSION THREAD

**Discussion by walsoo** [apohl79/audiogridder#282] [Q&A] [answered]
Hi! 

I first tried the 1.0 server, and then the latest release.

I initially set audiogridder to scan the whole VST folder. Scanning them took ages, so I terminated the server and dragged
acustica plugins into another folder. 

Despite checking and unchecking plugin boxes, and trying the latest release, I can't make it rescan the plugins.

Tried uninstallig both versions, but it seems to remember my VST folder as it is already set when I install a new one.

My server computer runs windows 7. Is there any file saved in regedit that needs to be deleted to rescan the VST's?

Thank you!

**Comment by apohl79** [comment, marked as answer]
From the tray menu you can start a rescan.

To reset an installation (w/o reinstalling), just wipe the config and cache files. See here for the location:

https://audiogridder.com/server/#config

**Reply by walsoo** [reply]
Thank you Andreas.

Wiped the config and rescanned the plugins.

Still, a lot of the acustica plugins (such as Azure) were not showing up. 
I then tried to move them up a couple of subfolders and rescan
and they were found by audiogridder!

Best
